conformance_tests/sysman/test_sysman_init/src/test_init_sysman.cpp

```diff
@@ -24,4 +24,51 @@ TEST(SysmanInitTests,
   ASSERT_GT(pCount, 0);
 }
 
+TEST(SysmanInitTests,
+     GivenCoreAndSysmanInitializedSysmanHandleWorksForEnumFrequencyDomanis) {
```

Comment:
Better to refactor the Init tests into separate executable for isolated testing
Similar to https://github.com/oneapi-src/level-zero-tests/commit/ab306e0813a9a7f7b5491a3cc3a6559262a56469